src/main/java/de/chojo/repbot/commands/Reputation.java

```diff
@@ -20,18 +19,19 @@
 import net.dv8tion.jda.api.entities.MessageEmbed;
 
 import javax.sql.DataSource;
-import java.awt.Color;
+import java.awt.*;
```

Comment:
IntelliJ just randomly forcing wildcard imports, I guess?

Reply:
yeah reverted this setting already. gonna push this boy back